Client/Frontend/Browser/TabTrayController.swift

```diff
@@ -402,6 +405,14 @@ class TabTrayController: UIViewController, Themeable {
         collectionView?.visibleCells.compactMap({ $0 as? TabCell }).forEach { $0.applyTheme(theme) }
         toolbar.applyTheme(theme)
     }
+    
+    /// Reset the empty private browsing state (hide the details, unhide the learn more button) if it was changed
+    func resetEmptyPrivateBrowsingView() {
+        if !emptyPrivateTabsView.detailsLabel.isHidden {
```

Comment:
Check not needed, just set their `isHidden` flags, no need to protect against it IMO.